Godeps/Godeps.json

```diff
@@ -348,11 +348,11 @@
 			"Rev": "95ba30457eb1121fa27753627c774c7cd4e90083"
 		},
 		{
-			"ImportPath": "github.com/masterzen/winrm/soap",
-			"Rev": "54ea5d01478cfc2afccec1504bd0dfcd8c260cfa"
+			"ImportPath": "github.com/masterzen/winrm",
+			"Rev": "ef3efbb97f99fc204bd9c7edf778a0dbd9781baf"
 		},
 		{
-			"ImportPath": "github.com/masterzen/winrm/winrm",
+			"ImportPath": "github.com/masterzen/winrm/soap",
 			"Rev": "54ea5d01478cfc2afccec1504bd0dfcd8c260cfa"
```

Comment:
Both `github.com/masterzen/winrm` and `github.com/masterzen/winrm/soap` should reference the same rev.